### PR TITLE
pua: treat 325 as false value

### PR DESCRIPTION
**Pull request by chenhengqi**
cc @flaneur2020  @ICEYSELF

### CODE REVIEW

**Review by flaneur2020 — APPROVED** [review]

File: src/evaluator/mod.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ impl Evaluator {
 
     fn is_truthy(obj: Object) -> bool {
         match obj {
-            Object::Null | Object::Bool(false) => false,
+            Object::Null | Object::Bool(false) | Object::Int(325) => false,
```
Comment (Artoria2e5) [inline src/evaluator/mod.rs:24]: 我觉得应该是以 3.5 分（350）为界去判定：
```suggestion
            Object::Null | Object::Bool(false) => false,
            Object::Int(x) => x > 350,
```

**Comment by rijieli** [comment]
325 is innocent, maybe a global variable RANK325 as False is better.

**Comment by chenhengqi** [comment]
I still prefer the idea of just let `325` be a false value, just like `三二五` in the language.

Feel free to ignore.

**Comment by Artoria2e5** [comment]
就这样吧。

**Pull request merged**